tests/integration/span_events/test_span_events_special_max_disabled.php

```diff
@@ -11,22 +11,22 @@
 newrelic.distributed_tracing_enabled = 1
 newrelic.transaction_tracer.threshold = 0
 newrelic.cross_application_tracer.enabled = false
-newrelic.special.max_span_events = 0
+newrelic.span_events.max_samples_stored = 0
 */
 
 /*EXPECT_SPAN_EVENTS
 [
   "?? agent run id",
   {
-    "reservoir_size": 1000,
-    "events_seen": 1000 
+    "reservoir_size": 10000,
+    "events_seen": 2000
   },
   "??"
 ]
  */
 
-$NEWRELIC_SPAN_EVENTS_MAX = 1000; // The agent internal maximum of span events 
-				  // per transaction.
+$NEWRELIC_SPAN_EVENTS_MAX = 3000; // The number of span events to send per transaction. 
```

Comment:
Why is this 3000?  Seems like 2000 would be following the original example but since it is a max maybe that isn't important?

Reply:
This is just the number of spans that we will attempt to send.  For this test, we want it to be greater than the configured number.  The tests are going to be expanded and extended in issue #365